marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/data-services/graph/nodeExpand.sjs

```diff
@@ -252,10 +265,19 @@ if (isConcept) {
       if(item.nodeCount) {
         hasRelationships = graphUtils.relatedObjHasRelationships(objectIRI, hashmapPredicate);
       }
+
+      let newLabel = getCustomLabel(entityType,  item.docRelated);
       let nodeExpanded = {};
       nodeExpanded.id = nodeId;
       nodeExpanded.docURI = nodeDocUri;
-      nodeExpanded.label = nodeLabel;
+
+      if (newLabel.toString().length === 0) {
+        nodeExpanded.label = nodeLabel;
```

Comment:
Can we update nodeLabel to have the value we want? And just assing values here. To keep a clean block of code here :)

Reply:
Done, thanks for your input @aniamy